src/css/application.css

```diff
@@ -566,7 +566,7 @@ body {
   position: absolute;
   top: 50%;
   left: 50%;
-  height: 20%;
+  height: 50%;
```

Comment:
Does this have any effect on the way it renders in the Gist import case? (Visual inspection would suggest no, but I can’t figure out why!)